### PR TITLE
Recreate audio_control example

### PR DESCRIPTION
**Pull request by GitGhillie**
## Description

Recreated the audio_control example from Bevy.

Run with `cargo run --example audio_control`. I didn't add `Windless Slopes.ogg` because I wasn't sure if I could just do that, so let me know what the preference is there.

### CODE REVIEW

**Comment by SolarLiner** [comment]
> I didn't add `Windless Slopes.ogg` because I wasn't sure if I could just do that

You bring a good point, and that makes me think that I added the drums asset to the examples without too much care for usage rights -- however since this crate is supposed to be upstreamed, it shouldn't be too much of an issue for Windless Slopes (and in fact all examples should use it instead). I definitely would ping a Bevy maintainer about it though!

File: examples/audio_control.rs
Comment (SolarLiner) [inline examples/audio_control.rs]: `sink` is a left-over vocabulary from `rodio` that is actually counter-intuitive to people that are already familiar with audio; "sinks" generally refer to outputs, instead of sources. The new names can be bikeshed but given we'll most probably rename all `*Handle` structs to `*Control` it makes more sense to name the variable similarly.

```suggestion
    if let Ok(mut control) = music_controller.get_single_mut() {
        if keyboard_input.just_pressed(KeyCode::Equal) {
            *target_volume += 0.1;
            error(control.set_volume(*target_volume + 1.0, Tween::default()));
        } else if keyboard_input.just_pressed(KeyCode::Minus) {
            *target_volume -= 0.1;
            error(control.set_volume(*target_volume + 1.0, Tween::default()));
        }
    }
```

**Review by SolarLiner — APPROVED** [review]

File: examples/audio_control.rs
Comment (SolarLiner) [inline examples/audio_control.rs]: Actually, missed that, you can use `AudioFileBundle` which is a simple type alias, but sets the type of `source` so that you don't have to specify it (similarly to how `PbrMaterialBundle` works)

```suggestion
        AudioFileBundle {
            source: asset_server.load("Windless Slopes.ogg"),
```

**Review by SolarLiner — CHANGES_REQUESTED** [review]

**Comment by SolarLiner** [comment]
It looks like adding bevy as a dev-dependency broke the doc test in [src/lib.rs:29](https://github.com/SolarLiner/bevy-kira-components/blob/main/src/lib.rs#L29), probably because the default features were turned back on. Not sure what's the best way to resolve it.

**Comment by GitGhillie** [comment]
Hmm pushing commits doesn't seem to trigger a re-run of the CI so it's hard to try out a fix for the test...

I'll ping a maintainer about the Windless Slopes question tomorrow!

**Comment by SolarLiner** [comment]
re. `Windless Slopes`, it's from [this pack](https://bakudas.itch.io/generic-rpg-pack), which has a CC0 1.0 license attached to it, so it's fine to include it here.

**Comment by SolarLiner** [comment]
> pushing commits doesn't seem to trigger a re-run of the CI so it's hard to try out a fix for the test...

Looks like as a "first-time contributor" I'll need to approve all your runs? At least for your next PR it might work better 🙃

**Comment by GitGhillie** [comment]
Ha interesting, I'll try to reproduce it locally in the meantime

**Comment by GitGhillie** [comment]
I think I found the solution. By using `no_run` the doc example will still build but it won't try to run it. Which I think is the intention anyways

**Review by SolarLiner — APPROVED** [review]

**Comment by GitGhillie** [comment]
Just in case you're waiting for me: I don't have a merge button 😛

**Comment by SolarLiner** [comment]
Ha, that's another thing I didn't know lol, it'd be nice if you could, I'll look into the settings.

**Comment by SolarLiner** [comment]
Well, I guess there's no other way than to have me be the one clicking on the button, so be it

**Pull request merged**
